src/server/utils/onModuleLoad.js

```diff
@@ -71,15 +72,10 @@ export function validateCspIsPresent(csp) {
   }
 }
 
-/* eslint complexity:  ['error', 12] */
-export default function onModuleLoad({
-  module,
-  moduleName,
-}) {
+export function setRootModuleConfigurations(module, moduleName) {
```

Comment:
this change should not be necessary with https://github.com/americanexpress/one-app/pull/984/files#diff-c2c760b479b24ea34c7eae704bba77b771c6ce7b48f7b4e6c8b386140de2ca9b

Reply:
But that is a draft PR thats expected to go in after this PR